content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -53,20 +74,27 @@ There are two types of filters:
 - **Custom filters** (highlighted in purple above): Filter commits by one or
   more of the following fields:
 
-  - Git related fields such as Git branch, commit message, tag and author
   - Column values (values of metrics, hyperparameters, etc.) and their deltas
+  - Git related fields such as Git branch, commit message, tag and author
+
+    <admon type="info">
 
-  <admon type="info">
+    The `Branch`filter displays only the specified branch and its commits.
 
-  The `Branch`filter displays only the specified branch and its commits.
+    On the other hand, the `Commits on branch` filter will also display branches
+    that contain any commits **in common** with the specified branch. This is
+    useful when
+    [some Git branches are nested inside other branches](#nested-branches),
+    because of which the nested branches are not displayed in the project table.
+    If you use the `Commits on branch = feature-branch-1` filter (where
+    `feature-branch-1` nested within `main`), commits from this branch are
+    included in the result. These commits appear within the branch into which
+    the nested branch was merged (i.e., `main`). A hint is present indicating
```

Comment:
😖 Sorry, this is still very confusing to read.
So, IIUC - if `feature-branch-1` is nested in `main` (=it is inactive, no new unique commits), AND I use `Commits on branch = feature-branch-1` all of `feature-branch-1` commits will appear within `main` and I'll have an empty `feature-branch-1` row?

Reply:
[Revised](https://github.com/iterative/dvc.org/pull/4758/commits/40e6b455f1d0e260370c22457b0051672c82d744). There'll be no `feature-branch-1` row.